client/CMakeLists.txt

```diff
@@ -0,0 +1,49 @@
+#
+# client/CMakeLists.txt
+# Written by Rylander and Mejborn, Team 1, DAT265.
+#
+#
+
+
+cmake_minimum_required (VERSION 3.16)
+project (SMCE_client)
+
+find_package (Threads REQUIRED)
+find_package (SMCE REQUIRED)
+
+set (SMCE_RES "${PROJECT_BINARY_DIR}/SMCE_Res")
+
+include (FetchContent)
+FetchContent_Declare (Termcolor
+        GIT_REPOSITORY "https://github.com/ikalnytskyi/termcolor"
+        GIT_TAG "v2.0.0"
+        GIT_SHALLOW On
+        GIT_PROGRESS On
+        )
+FetchContent_Declare (Lyra
+        GIT_REPOSITORY "https://github.com/bfgroup/Lyra"
+        GIT_TAG "1.5.1"
+        GIT_SHALLOW On
+        GIT_PROGRESS On
+        )
+FetchContent_MakeAvailable(Termcolor Lyra)
+
+include_directories(${PROJECT_SOURCE_DIR}/src)
```

Comment:
Never use global `include_directories`; see `target_include_directories` for a pure alternative